girder/api/v1/system.py

```diff
@@ -422,3 +404,58 @@ def stream():
                         break
                     yield data
         return stream
+
+    def _fixBaseParents(self, progress):
+        fixes = 0
+        user = self.getCurrentUser()
+        models = ['folder', 'item']
+        steps = sum(self.model(model).find().count() for model in models)
+        progress.update(
+            total=steps, current=0,
+            title='Checking for incorrect base parents (Step 2 of 3)')
```

Comment:
and then take in the `(Step 2 of 3)` text in this function as a param which is concatenated here.

If some arbitrary caller calls this function, it may not be step 2 and it may not be out of 3 steps total.